src/java/org/apache/cassandra/cql3/restrictions/StatementRestrictions.java

```diff
@@ -485,7 +485,7 @@ public boolean isTopK()
      * @param kind the column type
      * @return the <code>Restrictions</code> for the specified type of columns
      */
-    private Restrictions getRestrictions(ColumnMetadata.Kind kind)
+    public Restrictions getRestrictions(ColumnMetadata.Kind kind)
```

Comment:
the method is only referenced within the class. It is unnecessary to open the access to `public`